.circleci/config.yml

```diff
@@ -94,9 +94,23 @@ jobs:
           name: "Check that there are no generated changes"
           command: make check-no-changes
 
+  deploy_test:
+    executor: ubuntu_vm
+    environment:
+      NUODB_CP_URL_BASE: "https://dev.us-east-1.dbaas.nuodb.io/api"
```

Comment:
Can you also make this a CircleCI environment variable so that we do not have to make a code change to use a different URL? We also don't want to expose the URL of the service in a public repo.

Reply:
Sure